app/main.py

```
@@ -1 +1,15 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "cp":
+        return
+    src_file = parts[1]
+    dest_file = parts[2]
+    if src_file == dest_file:
+        return
+    if not os.path.exists(src_file):
+        return
```

Similar to the previous comment, it might be helpful to provide feedback or raise an exception if the source file does not exist, instead of silently returning.